pkg/build/builder/dockerutil.go

```diff
@@ -17,11 +19,11 @@ type DockerClient interface {
 
 // pushImage pushes a docker image to the registry specified in its tag
 func pushImage(client DockerClient, name string, authConfig docker.AuthConfiguration) error {
+	glog.Infof("Pushing %s", name)
```

Comment:
Initially I wrote it with dots but since we're not using it too often I
removed them, per your request they're back.

On 10:41pm, Mon, Mar 30, 2015 Clayton Coleman <notifications@github.com>
wrote:

> In pkg/build/builder/dockerutil.go
> <https://github.com/openshift/origin/pull/1498#discussion_r27429699>:
>
> > @@ -17,11 +19,11 @@ type DockerClient interface {
> >
> >  // pushImage pushes a docker image to the registry specified in its tag
> >  func pushImage(client DockerClient, name string, authConfig docker.AuthConfiguration) error {
> > +	glog.Infof("Pushing %s", name)
>
> Pushing %s ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/1498/files#r27429699>.
>